README.md

```diff
@@ -107,23 +107,12 @@ Con un semplice Click permette di Passare da un profilo di Firefox all'altro!
 
 ### Altri progetti Mozilla
 
-Sono numerosi i progetti portati avanti da Mozilla, ma sicuramente quelli elencati qui sotto sono i più importanti e conosciuti. Tra quelli che non ho citato ci sono anche Firefox for Android, Mozilla Sync, il progetto WebMaker, Mozilla Science, Mozilla-Knights OpenNews, Mozilla Advocacy; come potrai notare facendo qualche ricerca, si rivolgono a persone con molte competenze diverse (insegnanti, scienziati, giornalisti).
+Numerosi sono i progetti portati avanti dalla fondazione Mozilla, e tra quelli non elencati ci sono progetti del calibro di Mozilla Sync, WebMaker, Mozilla Science, Mozilla-Knights OpenNews, Mozilla Advocacy, Mozilla Winter of Security. Tutti questi progetti si rivolgono a persone con differenti competenze (ad esempio: Insegnanti, Scienziati, Giornalisti e Studenti) e sono legati da un unico comune denominatore: la visione del web come risorsa libera, aperta a tutte le popolazioni del mondo. In Dettaglio tra i progetti di Mozilla troviamo il Programma di Posta Thunderbird.
```

Comment:
Mozilla Foundation invece di fondazione Mozilla, Mozilla Sync lo toglierei é un progetto minore.
Mentre Winter of security e Advocacy non sono progetti ma rispettivamente una campagna e team.
Come progetti metterei Rust, MDN e MozVR.
Scriverei poi Insegnanti ecc in minuscolo, "unico comune denominatore" é troppo complesso lo semplificherei.
Il "Programma di Posta" é sbagliato é un client email, altrimenti sembra un programma televisivo.